### PR TITLE
ZH-252 - 1+MG DAC Recommendation Approval Process

### PR DESCRIPTION
**Pull request by M-casado**
## Summary
Addition of the Node-specific SOP ``1+MG DAC Recommendation Approval Process``

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New content (non-breaking change which adds new content)
- [ ] Modified content (non-breaking change which modifies existing content)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Motivation and Context
SOP was ready to be added to the GH repo after internal reviews.

### References
[ZenHub #252](https://app.zenhub.com/workspaces/t43-gdi-sops-662a0b0fe38b0405fb600d67/issues/zh/252)

## Changes Introduced
- Addition of new SOP ``1+MG DAC Recommendation Approval Process`` at ``sops/european-level/GDI-SOP0003_SOP-1+MG-DAC-Recommendation-Approval``

## Review
It was reviewed internally to T4.3, and also by Regina Becker.

## Additional Notes
I made changes to the **content** of the SOP as well, including new diagram and steps, for clarity. See [this commit](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/pull/34/commits/64fe8f3256030d2fa8cbcffd85eddf1707900692) to know what changed.

## Checklist:
**General Compliance:**
- [x] My changes follow the code style of this project ([GDI SOP Style Guide](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_style-guide.md)) and the file naming conventions of the [file accessioning proposal](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs//GDI-SOP_sop-accessioning.md).
- [x] I have verified that all new updated content is accessible, including checking that all external references are readable (i.e., no broken links). These may include references to external resources that should be resolvable, and internal references among SOPs.
- [x] I have properly added this PR's changes to the repository [CHANGELOG.md](../CHANGELOG.md).

Only applicable if the PR includes new, or changes to, GDI SOPs (i.e., documents at ``sops/``):
- [x] My SOP-related changes adhere to the [Generic SOP Template](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_sop-template.md), including format and required fields.
- [x] I have consulted the [Charter](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_charter.md), [ISM](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_information-service-management.md), and [ORR](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/blob/main/docs/GDI-SOP_organisational-roles-and-responsibilities.md) documents to ensure compliance. 
- [x] I am complying with the established procedure for SOP creations and modifications, including respecting review phases and notifying needed contributors for reviews.

### CODE REVIEW

**Comment by M-casado** [comment]
Important for all reviewers and especially @jdylan: I not only transformed the document to markdown, I also added a few **modifications to the procedures and a full diagram**, for clarity. To ease the review, focus on [this commit](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/pull/34/commits/64fe8f3256030d2fa8cbcffd85eddf1707900692), which contains all new additions (in content, not just format) to the procedure of the SOP.

File: sops/european-level/GDI-SOP0003_SOP-1+MG-DAC-Recommendation-Approval.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    B5 -->|Recommendation Upheld| B6
+    B5 -->|Recommendation Overruled| B7
+    B5 -->|Review Delayed| B8
+    B1 -->|Application<br>is rejected| B10
+    B7 --> B10
+    B8 -->|Follow up<br> review process| B5
+````
+
+### 8. Procedure
+#### 1. Initiate application check
```
Comment (GabiRinck) [inline sops/european-level/GDI-SOP0003_SOP-1+MG-DAC-Recommendation-Approval.md:99]: I think there were comments about consistent & clear / unambiguous numbering previously in a different SOP (the ISM one?). both ### headers & #### headers are both numbered with 1., 2., 3. etc. I think we need to come up with a system that we apply across all SOPs, if possible. Here, what about using (1), (2), (3) etc for the #### headers??

File: sops/european-level/GDI-SOP0003_SOP-1+MG-DAC-Recommendation-Approval.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+If the **NCP agrees** with the recommendation, then **confirm the approval of the data access request**: move to **[step 9](#9-inform-data-user-approval-confirmed)**.
+
+#### 5. Initiate NCP review process
+| Step identifier | When                                           | Who      |
+|-----------------|------------------------------------------------|----------|
+| 5               | DATE:A has not been passed, and the NCP disagrees with the 1+MG DAC recommendation | 1+MG DAC |
+
+If the NCP disagrees with the recommendation, **initiate the review process for handling NCP disagreements** as per SOP [NCP review process of disagreement with 1+MG DAC recommendation](https://github.com/GenomicDataInfrastructure/standard-operating-procedures/issues/33). This process must be completed by the time limit set in that SOP (``DATE:B``).
+
+Depending on the outcome of the NCP review SOP:
```
Comment (GabiRinck) [inline sops/european-level/GDI-SOP0003_SOP-1+MG-DAC-Recommendation-Approval.md:155]:
```suggestion
Depending on the outcome of the NCP review process:
```

File: sops/european-level/GDI-SOP0003_SOP-1+MG-DAC-Recommendation-Approval.md [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+#### 9. Inform data requestor (Approval confirmed)
+| Step identifier | When                                           | Who      |
+|-----------------|------------------------------------------------|----------|
+| 9               | After confirmation of approval is reached      | 1+MG DAC |
+
+Regardless of the path of confirmation, once the approval has been confirmed:
+- **Record the decision and reasons**.
+- **Inform the data requestor**.
+
+This SOP is considered finished at this point.
```
Comment (GabiRinck) [inline sops/european-level/GDI-SOP0003_SOP-1+MG-DAC-Recommendation-Approval.md:192]:
```suggestion
The process is considered finished at this point.
```

File: sops/european-level/GDI-SOP0003_SOP-1+MG-DAC-Recommendation-Approval.md [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+#### 10. Inform data requestor (Rejection confirmed)
+| Step identifier | When                                           | Who      |
+|-----------------|------------------------------------------------|----------|
+| 10               | After rejection of request is reached          | 1+MG DAC |
+
+Regardless of the path of rejection:
+- **Record the decision and reasons** (e.g., missing documents, invalid data, non-compliance with ethical standards...). Include options to rectify these issues for the data requestor to amend and re-submit the application.
+- **Inform the data requestor**.
+
+This SOP is considered finished at this point.
```
Comment (GabiRinck) [inline sops/european-level/GDI-SOP0003_SOP-1+MG-DAC-Recommendation-Approval.md:203]:
```suggestion
The process is considered finished at this point.
```

**Review by GabiRinck — COMMENTED** [review]
Nicely complements the SOP about the NCPs' veto.
Just a few minor comments added.
Many thanks.

**Pull request merged**